python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -345,6 +345,7 @@ def __call__(self, index):
 
     def __getitem__(self, index):
         """Return a subjob based upon index"""
+        logger.debug("__getitem__")
```

Comment:
Given it's a call to the ```__getitem__``` which comes from this class I would have thought there is enough information in the logging to be able to get back here quite quickly. I can drop it seeing how there is a debug at the top of ```_getItem``` anyway which is explicitly called.